include/sys/fs/zfs.h

```diff
@@ -475,8 +479,9 @@ typedef enum {
 #define	ZPL_VERSION_3			3ULL
 #define	ZPL_VERSION_4			4ULL
 #define	ZPL_VERSION_5			5ULL
-#define	ZPL_VERSION			ZPL_VERSION_5
-#define	ZPL_VERSION_STRING		"5"
+#define	ZPL_VERSION_6			6ULL
+#define	ZPL_VERSION			ZPL_VERSION_6
+#define	ZPL_VERSION_STRING		"6"
```

Comment:
Sure, I will get rid of this change on the next revision. Meanwhile, please inspect the rest of this patch if you've got some time. Thanks.